settings.py

```diff
@@ -17,4 +17,4 @@
 
 USE_TZ = False
```

Comment:
Setting `USE_TZ` to `False` means that Django will not use timezone-aware datetimes. Consider setting it to `True` to handle timezones properly, especially if the application will be used in multiple time zones.